dist/flight/flight.service.d.ts

```diff
@@ -0,0 +1,9 @@
+import { Flight } from "./flight.model";
+export declare class FlightService {
+    flight: Flight[];
+    bookFlight(from: string, to: string, date: string, price: number, seats: number, company: string): void;
```

Comment:
bookFlight or cancelFlight, should return success/failure , failure can be some business logic validations  including any handled exceptions